src/main/java/thePackmaster/cards/summonspack/Leprechaun.java

```diff
@@ -60,5 +60,6 @@ public List<TooltipInfo> getCustomTooltips() {
     @Override
     public void upp() {
         upMagic(UPGRADE_MAGIC);
+        uDesc();
```

Comment:
Updating the description should be handled by `AbstractPackmasterCard` as long as an upgrade description is defined, so I don't think you need this call.